cla_frontend/assets-src/javascripts/app/partials/case_detail.alternative_help.html

```diff
@@ -40,7 +40,11 @@ <h1 class="u-pullLeft">Alternative help</h1>
       <div class="Grid">
         <div class="Grid-row cf">
           <div class="Grid-col Grid-col--1-3">{{ ::provider.address }}</div>
-          <div class="Grid-col Grid-col--1-3" ng-bind-html="::provider.helpline|escapeHtml|nl2br"></div>
+          <div class="Grid-col Grid-col--1-3" ng-If="provider.telephone_numbers">
```

Comment:
Can you also prefix the scope variable with `::`. It prevents it being a two-way binding and creating too many watches